addons/nuxeo-template-rendering/nuxeo-template-rendering-core/src/main/java/org/nuxeo/template/XMLSerializer.java

```diff
@@ -20,143 +20,38 @@
 
 package org.nuxeo.template;
 
-import java.text.ParseException;
-import java.text.SimpleDateFormat;
-import java.util.ArrayList;
 import java.util.List;
 
-import org.apache.commons.logging.Log;
-import org.apache.commons.logging.LogFactory;
-import org.dom4j.Attribute;
-import org.dom4j.Document;
 import org.dom4j.DocumentException;
-import org.dom4j.DocumentFactory;
-import org.dom4j.DocumentHelper;
-import org.dom4j.Element;
-import org.dom4j.Namespace;
-import org.dom4j.QName;
-import org.dom4j.tree.DefaultElement;
-import org.nuxeo.ecm.core.api.DocumentModel;
-import org.nuxeo.template.api.InputType;
+import org.nuxeo.runtime.api.Framework;
 import org.nuxeo.template.api.TemplateInput;
+import org.nuxeo.template.serializer.service.SerializerService;
+import org.nuxeo.template.serializer.executors.Serializer;
 
 /**
- * {@link TemplateInput} parameters are stored in the {@link DocumentModel} as a single String Property via XML
- * Serialization. This class contains the Serialization/Deserialization logic.
+ * Should use now Framework.getService(SerializerService.class).getSerializer("xml")
  *
  * @author Tiry (tdelprat@nuxeo.com)
+ * @author bjalon (bjalon@qastia.com)
  */
+@Deprecated(since = "11.1")
```

Comment:
Also add a `@deprecated` tag in the Javadoc: `@deprecated since 11.1, use XYZ instead`

Reply:
Hi,

I don't know how to modify the javadoc. Other point when you say "XYZ", you mean 11.1.0 ? because if I'm looking for most of the depreciation for the 11.X.X you added, you put 11.1 (example [MimetypeIconUpdater.java](https://github.com/nuxeo/nuxeo/blob/457a1b98ae32d1ced0a4de0009b979214da69b52/nuxeo-services/nuxeo-platform-filemanager-core-listener/src/main/java/org/nuxeo/ecm/platform/filemanager/core/listener/MimetypeIconUpdater.java#L65)).

Reply:
do I put for removal ?